xendit_pos/utils/qrcode.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from ast import In, Or
+import base64
+from operator import or_
+import qrcode
+import io
+
+class Qrcode():
+
+    def initQrCode(self):
+        return qrcode.QRCode(  
+            version = 1,
+            error_correction = qrcode.constants.ERROR_CORRECT_L,
+            box_size = 10,
+            border = 4,
+        )  
+
+    def generateQrCode(self, xendit_invoice_url):
+        qrcode = self.initQrCode()
+        qrcode.add_data(xendit_invoice_url)
+        qrcode.make(fit = True)
+        qrcode_image = qrcode.make_image(fill_color = "black", back_color = "transparent")
```

Comment:
😬 let's keep the changes of single and double quote consistent in the changes.

Reply:
:D forgot it